content/en/blog/signal-boosting.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Signal Boosting
+date: 2022-09-24
+---
```

Comment:
```suggestion
authors:
  - Joachim Viide
---
```
[Marvin's earlier PR](#916) adds support for specifying authors and [he's conveniently already added your details](https://github.com/preactjs/preact-www/blob/69398eda85599551cfc04df82889259a44d05f22/src/config.json#L512-L515).

Apologies if anyone else also contributed, I'm unaware.

Reply:
Thanks! 👍